src/tgbot/senders/next_message.py

```diff
@@ -36,17 +35,24 @@ async def next_message(
         asyncio.create_task(check_queue(user_id))
         # TODO: also edit / delete
         return await send_queue_preparing_alert(user_id)
-    
-    send_new_message = prev_reaction_id is None or Reaction(prev_reaction_id).is_positive
+
+    send_new_message = (
+        prev_reaction_id is None or Reaction(prev_reaction_id).is_positive
+    )
     if not send_new_message and prev_update_can_be_edited_with_media(prev_update):
         msg = await prev_update.callback_query.message.edit_media(
             media=get_input_media(meme),
             reply_markup=meme_reaction_keyboard(meme.id),
         )
+        meme_caption_with_referral_link = (f"{meme.caption}\n\n" if meme.caption else "") + f'<a href="https://t.me/ffmemesbot?start=s_{user_id}_{meme.id}">FastFoodMemes</a>'
```

Comment:
1. move function to utils.py
2. fix code line length codestyle
3. hardcoded "ffmemesbot" is not ok for local / staging / production environment testing
4. is "s_" a good option?
5. have you tested that this deeplink value appears in users table? What if a user was already created?

Reply:
1. done
2. done
3. Put that into `.env`
4. `s_` had been used in legacy ffmemesbot code, stands for "share".
5. In case if we see `/start` from a created user, we update only user-changable fields. Like username, first/last name, is_premium, etc.

Reply:
3.  `bot.application.bot.username` - does it really stores bot's username? Cool finding if that's true